tools/flaky_test_monitor/helpers/level1.go

```diff
@@ -0,0 +1,42 @@
+package helpers
+
+import (
+	"time"
+)
+
+// models single line from "go test -json" output
+type RawTestStep struct {
```

Comment:
I don't usually consider necessary data structures as "helpers" per se. I'd be okay with these living next to the functions but in a different file.

Reply:
The issue is that there are multiple `main` packages and we need to reuse those data structures between level 1 and 2. I can't put them in the same folder as the main functions because they aren't reference-able by another package (e.g. level 2 can't reference level 1 data structures if they're in the main package).

I renamed the `helpers` package to `common` to make it clearer that they're not helpers but used in common between levels 1 and 2

Reply:
https://github.com/onflow/flow-go/pull/1707/commits/ebd163c35b9499bb10d45698de196e18f21a6e40